index.html

```diff
@@ -0,0 +1,97 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <title>CV - Pinchuk Vladislav</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style type="text/css">
```

Comment:
Looks like type attr unneeded in this case. Fill free to use [HTML validator](https://validator.w3.org/#validate_by_input)